### PR TITLE
https://issues.redhat.com/browse/ACM-12941

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -52,6 +52,8 @@ For other {global-hub} topics, see link:../global_hub/global_hub_overview.adoc#m
 [#observability-whats-new]
 == Observability
 
+* For more environment stability with default settings, the default CPU request is increased to `500m` and memory request is increased to `1Gi` for the `observability-thanos-compact`. See link:../observability/obs_config.adocobservability-pod-capacity-requests[Observability pod capacity requests] for more details.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* For more environment stability with default settings, the default CPU request is increased to `500m` and memory request is increased to `1Gi` for the `thanos-compact` pod. See link:../observability/obs_config.adocobservability-pod-capacity-requests[Observability pod capacity requests] for more details.
```

File: observability/obs_config.adoc [hunk 2]
```diff
@@ -165,8 +165,8 @@ Observability components require 2701mCPU and 11972Mi memory to install the obse
 
 | observability-thanos-compact
 | thanos-compact
-| 100
-| 512
+| 500
+| 1Gi
```
Comment (philipgough) [inline observability/obs_config.adoc]: i think this needs to be changed to 1024

File: observability/obs_config.adoc [hunk 3]
```diff
@@ -165,8 +165,8 @@ Observability components require 2701mCPU and 11972Mi memory to install the obse
 
 | observability-thanos-compact
 | thanos-compact
-| 100
-| 512
+| 500
```
Comment (philipgough) [inline observability/obs_config.adoc:168]: this value is good

File: observability/obs_config.adoc (hunk 2 above)
Comment (dockerymick) [inline observability/obs_config.adoc]:
```suggestion
| 1024
```

File: release_notes/whats_new.adoc [hunk 4]
```diff
@@ -52,6 +52,8 @@ For other {global-hub} topics, see link:../global_hub/global_hub_overview.adoc#m
 [#observability-whats-new]
 == Observability
 
+* For more environment stability with default settings, the default CPU request is increased to `500m` and memory request is increased to `1Gi` for the `thanos-compact` pod. See link:../observability/obs_config.adocobservability-pod-capacity-requests[Observability pod capacity requests] for more details.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* For more environment stability with default settings, the default CPU request is increased to `500m` and memory request is increased to `1024Mi` for the `thanos-compact` pod. See link:../observability/obs_config.adocobservability-pod-capacity-requests[Observability pod capacity requests] for more details.
```

**Comment by dockerymick** [comment]
@philipgough thanks for your review! I updated your suggestions. @Xiang-Yin-ACM when you have a chance, will you also review the PR?

**Review by swopebe — APPROVED** [review]

**Review by Xiang-Yin-ACM — APPROVED** [review]
Looks good for me

File: observability/obs_config.adoc (hunk 2 above)
Comment (quchangl-github) [reply observability/obs_config.adoc]: @philipgough I checked in the compact pod, the requested memory is 1Gi, should we keep them consistent？

```
    resources:
      requests:
        cpu: 500m
        memory: 1Gi
```
@dockerymick FYI.

Comment (philipgough) [reply observability/obs_config.adoc]: @quchangl-github - the header for that column on the table is `Memory (Mi)` so 1024 is the equiv of 1Gi

**Review by quchangl-github — APPROVED** [review]

**Comment by dockerymick** [comment]
@quchangl-github thanks for your comment and suggestion. I assume the user will know the conversion? I'm not sure tho. @philipgough thanks for answering the comment. Does the PR look good to you?

**Review by philipgough — APPROVED** [review]
/lgtm

**Comment by dockerymick** [comment]
Thanks everyone for the reviews and approvals! Merging this PR

**Pull request merged**